docs/bb-theme/customizer-settings/general.md

```diff
@@ -83,7 +83,15 @@ See [this article about how Font Awesome icons load](/bb-theme/defaults-for-layo
 
 ### Theme Medium Breakpoint and Mobile Breakpoint
 
-The number of pixels below which medium device behavior starts (for the Medium Breakpoint setting) or small device behavior starts (for the Mobile Breakpoint setting). Defaults are 992px and 768px, respectively. These settings affect theme behavior only with respect to header and nav layouts. For details about breakpoints and how they affect behavior in the Theme, [see this article](/beaver-builder/layouts/responsive-design/breakpoints-for-device-sizes.md).
+A breakpoint is the number of pixels of browser width below  which medium-device behavior or small-device behavior starts. Defaults are 992px for Medium and 768px, respectively. 
+
+These settings affect Theme behavior with respect to header and nav layouts. 
+
+For [theme headers](/bb-theme/customizer-settings/header.md#header-layout), the Nav Right, Nav Left, and Nav Centered + Inline Logo change to Nav Centered (menu below the logo) on medium and small devices. 
+
+For [navigation layouts](/bb-theme/customizer-settings/header.md#nav-layout), the Nav menu collapses to a Hamburger icon or MENU link on small devices. 
+
+The Beaver Builder plugin also has default breakpoints to control respoonsive behavior in content areas and in Themer layouts, which differ by one pixel. For a description of the difference, see [the article on Beaver Builder breakpoints](/beaver-builder/layouts/responsive-design/breakpoints-for-device-sizes.md). See also the sections on [how to use the mobile nav icon instead of a full menu at all device sizes](/beaver-builder/layouts/responsive-design/breakpoints-for-device-sizes.md#use-the-mobile-nav-for-larger-devices) and [how to keep the theme's header layout the same on large and medium devices](/beaver-builder/layouts/responsive-design/breakpoints-for-device-sizes.md#preserve-the-themes-header-layout-on-medium-devices).
```

Comment:
Spelling: `respoonsive` line 94

Reply:
Thank you! Fixed and committed.  It's very helpful to have the broken links show up in the local version, at least I can avoid that by the time I get to a PR.